terasoluna-tourreservation-domain/src/test/resources/test-context.xml

```diff
@@ -20,7 +20,27 @@
     </constructor-arg>
   </bean>
 
-  <bean id="passwordEncoder" class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
+  <bean id="passwordEncoder" class="org.springframework.security.crypto.password.DelegatingPasswordEncoder">
+    <constructor-arg name="idForEncode" value="pbkdf2" />
+    <constructor-arg name="idToPasswordEncoder">
+      <map>
+        <entry key="pbkdf2">
+          <bean class="org.springframework.security.crypto.password.Pbkdf2PasswordEncoder" />
+        </entry>
+        <entry key="bcrypt">
+          <bean class="org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder" />
+        </entry>
+        <!-- When using commented out PasswordEncoders, you need to add bcprov-jdk15on.jar to the dependency.
+        <entry key="argon2">
+          <bean class="org.springframework.security.crypto.argon2.Argon2PasswordEncoder" />
+        </entry>
+        <entry key="scrypt">
+          <bean class="org.springframework.security.crypto.scrypt.SCryptPasswordEncoder" />
+        </entry>
+        -->
+      </map>
+    </constructor-arg>
+  </bean>
```

Comment:
You can't actually use this `passwordEncoder`.

https://github.com/terasolunaorg/terasoluna-tourreservation/blob/2de9b51a6d32d0fe0b6dfba6e85369a39e935f8e/terasoluna-tourreservation-domain/src/test/java/org/terasoluna/tourreservation/domain/service/customer/CustomerServiceImplTest.java#L45-L53

Reply:
fix to use bean DelegatingPasswordEncoder
5d6190af7c1a0b413dfa58d97b2bcd0063f321bd